admin/class-admin-apple-themes.php

```diff
@@ -159,6 +159,28 @@ public static function render_field( $theme, $option_name ) {
 		);
 	}
 
+	/**
+	 * Return a Theme object loaded with db values based on the theme name string.
+	 *
+	 * @param string $theme_str The name of theme.
+	 * 
+	 * @return \Apple_Exporter\Theme|null $theme_object A Theme object loaded with db config,
+	 * or null if theme name not in registry.
+	 */
+	public static function get_loaded_theme( $theme_str ) {
```

Comment:
Recommend changing this function name to be more generic. It could be used in other places. There are a lot of locations in code right now where we're doing the pattern of create new theme, set name, load, and this function could abstract that.
```suggestion
	public static function get_theme_by_name( $theme_str ) {
```

Reply:
Nice, I like this name better.